app/client/utils.js

```diff
@@ -280,7 +281,7 @@ export const get_zero_filled_values = (key, value_array, date_array) => {
 
 export const processFiltersForQuery = filters => {
   let encoded_filters = encodeURIComponent("");
-  let delimiter = ":";
+  let delimiter = "|";
```

Comment:
created filter on accounts/transfer page uses `:` in encoded filter which causes URL error on load. This changes the delimter which doesn't have any impact across the app

Reply:
pipes aren't web safe